src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Compiler/PropertyInfoPassTest.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Symfony\Bundle\FrameworkBundle\Tests\DependencyInjection\Compiler;
 
+use Prophecy\Argument;
```

Comment:
yes (since PHPunit 4 IIRC). However, I think it's better to be consistent and either use PHPunit's mock system everywhere in the code base, or Prophecy/another one. Mixing them makes contributing harder (one has to learn both PHPunit mocks and Prophecy).